### PR TITLE
Updated start page created out of the template

### PR DESCRIPTION
**Pull request by harmony7**
This PR updates the start page created out of the template.

This is the previous page:

Screenshot 2023-02-20 at 16 51 13

The new start page looks like this:

Screenshot 2023-02-20 at 16 45 09

I'm not a graphic artist but I wanted to run the text by the team and get any opinions.

### CODE REVIEW

**Comment by harmony7** [comment]
Kailan suggested adding margins. I like it, it looks like this now:

Screenshot 2023-02-22 at 16 30 39

File: packages/remix-template/app/routes/index.tsx [hunk 1]
```diff
@@ -1,23 +1,31 @@
...
   return (
     <div style={{ fontFamily: "system-ui, sans-serif", lineHeight: "1.4" }}>
-      <h1>Welcome to Remix</h1>
+      <h1>Congratulations!</h1>
+      <p>
+        You’ve successfully created your <a target="_blank" href="https://remix.run/">Remix</a> project for <a target="_blank" href="https://developer.fastly.com/learning/compute/javascript/">Fastly Compute@Edge</a>.
+      </p>
+      <p>
+        If you already have a Remix application that you'd like to move to Compute@Edge, see
+        our <a target="_blank" href="https://github.com/fastly/remix-compute-js/blob/main/MIGRATING.md">Migration Guide</a>.
```
Comment (chrispoole643) [inline packages/remix-template/app/routes/index.tsx:16]: Would suggest lowercase for _migration guide_.

File: packages/remix-template/app/routes/index.tsx [hunk 2]
```diff
@@ -27,6 +35,18 @@ export default function Index() {
           </a>
         </li>
       </ul>
+      <p>
+        The Compute@Edge Adapter for Remix is provided as part of <a target="_blank" href="https://developer.fastly.com/labs/">Fastly Labs</a>&mdash;where Edge computing meets the cutting edge.
```
Comment (chrispoole643) [inline packages/remix-template/app/routes/index.tsx:39]: Would suggest a colon over the em-dash here, and lowercase _Edge_.

**Review by chrispoole643 — APPROVED** [review]
Super-minor grammar suggestions that could well just be my preference, but otherwise looks great 😃

File: packages/remix-template/app/routes/index.tsx (hunk 1 above)
Comment (harmony7) [reply packages/remix-template/app/routes/index.tsx:16]: ok

File: packages/remix-template/app/routes/index.tsx (hunk 2 above)
Comment (harmony7) [reply packages/remix-template/app/routes/index.tsx:39]: ok

**Comment by harmony7** [comment]
@chrispoole643 Thanks for suggestions, they make sense and I've incorporated them.

**Pull request merged**